pkg/block/index.go

```diff
@@ -282,7 +283,9 @@ func GatherIndexIssueStats(logger log.Logger, fn string, minTime int64, maxTime
 		}
 		l0 := lset[0]
 		for _, l := range lset[1:] {
-			if l.Name <= l0.Name {
+			if l.Name == l0.Name {
```

Comment:
It only makes sense if l.Value == l0.Value as well I guess.

Reply:
But we would hit the case that with same label name and different value. I think it has been not make sense with duplicate label name.